tribe-ext-relabeler.php

```diff
@@ -0,0 +1,322 @@
+		add_filter( 'tribe_event_label_singular_lowercase', [ $this, 'get_event_single_lowercase' ] );
+		add_filter( 'tribe_event_label_plural', [ $this, 'get_event_plural' ] );
+		add_filter( 'tribe_event_label_plural_lowercase', [ $this, 'get_event_plural_lowercase' ] );
+
+		// Venues.
+		add_filter( 'tribe_venue_label_singular', [ $this, 'get_venue_single' ] );
+		add_filter( 'tribe_venue_label_singular_lowercase', [ $this, 'get_venue_single_lowercase' ] );
+		add_filter( 'tribe_venue_label_plural', [ $this, 'get_venue_plural' ] );
+		add_filter( 'tribe_venue_label_plural_lowercase', [ $this, 'get_venue_plural_lowercase' ] );
+
+		// Organizers.
+		add_filter( 'tribe_organizer_label_singular', [ $this, 'get_organizer_single' ] );
+		add_filter( 'tribe_organizer_label_singular_lowercase', [ $this, 'get_organizer_single_lowercase' ] );
+		add_filter( 'tribe_organizer_label_plural', [ $this, 'get_organizer_plural' ] );
+		add_filter( 'tribe_organizer_label_plural_lowercase', [ $this, 'get_organizer_plural_lowercase' ] );
+	}
+
+	/**
+	 * Check if we have a sufficient version of PHP. Admin notice if we don't and user should see it.
+	 *
+	 * @link https://theeventscalendar.com/knowledgebase/php-version-requirement-changes/ All extensions require PHP 5.6+.
+	 *
+	 * @return bool
+	 */
+	private function php_version_check() {
+		$php_required_version = '5.6';
+
+		if ( version_compare( PHP_VERSION, $php_required_version, '<' ) ) {
+			if (
+				is_admin()
+				&& current_user_can( 'activate_plugins' )
+			) {
+				$message = '<p>';
+				$message .= sprintf( __( '%s requires PHP version %s or newer to work. Please contact your website host and inquire about updating PHP.', PLUGIN_TEXT_DOMAIN ), $this->get_name(), $php_required_version );
+				$message .= sprintf( ' <a href="%1$s">%1$s</a>', 'https://wordpress.org/about/requirements/' );
+				$message .= '</p>';
+
+				tribe_notice( PLUGIN_TEXT_DOMAIN . '-php-version', $message, [ 'type' => 'error' ] );
+			}
+
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Use Tribe Autoloader for all class files within this namespace in the 'src' directory.
+	 *
+	 * @return Tribe__Autoloader
+	 */
+	public function class_loader() {
+		if ( empty( $this->class_loader ) ) {
+			$this->class_loader = new Tribe__Autoloader;
+			$this->class_loader->set_dir_separator( '\\' );
+			$this->class_loader->register_prefix(
+				\Tribe\Extensions\Relabeler\NS,
+				__DIR__ . DIRECTORY_SEPARATOR . 'src'
+			);
+		}
+
+		$this->class_loader->register_autoloader();
+
+		return $this->class_loader;
+	}
+
+	/**
+	 * Gets the label from the database and caches it
+	 *
+	 * @param $key     string Option key for the label.
+	 * @param $default string Value to return if none set.
+	 *
+	 * @return string|null
+	 */
+	public function get_label( $key, $default = null ) {
+
+		$key = $this->get_options_prefix() . "_" . $key;
+
+		if ( ! isset( $this->label_cache[ $key ] ) ) {
```

Comment:
To make it a bit more explicit, plus removes the need to use a `!` operator.

```suggestion
		if ( empty( $this->label_cache[ $key ] ) ) {
```